movement-migration/post-move2-upgrade/scripts/initialize_partial_voting.move

```diff
@@ -0,0 +1,13 @@
+script {
+    use aptos_framework::aptos_governance;
+    use aptos_framework::governed_gas_pool;
```

Comment:
how about moving this script to the test_proposal/scripts folder? I think we probably don't want to enable it immediately after the L1 migration.